packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -213,11 +218,19 @@ module.exports = React.createClass({
         key="previewContainer"
         className={this.getPreviewClasses()}
       >
-        {this.loadChild('preview')}
+        {this.renderComponent()}
       </div>
     );
   },
 
+  renderComponent() {
+    return _.reduce(this.props.proxies, (accumulator, proxy) =>
+      React.createElement(proxy.component, _.assign({}, this.props, {
+        children: React.Children.only(accumulator),
+        ref: proxy.ref,
```

Comment:
I think this is a bad move going forward. [String refs are almost deprecated](https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute), so we should at least use callback refs. But callback refs also don't make sense here, because the CP shouldn't be aware of what proxies it is being passed, otherwise we might as well embed them inside. I suggest we only keep component classes/functions in the list of proxies. 

Is there any other reason besides tests to keep them? There must be a different way to check that the proxies are being loaded. E.g. Checking that their dom node(s) include the dom node(s) of the next proxy.